src/config.rs

```diff
@@ -1513,14 +1513,22 @@ impl Config {
 
     #[cfg(windows)]
     pub fn installed_config<'a>() -> Option<Cow<'a, Path>> {
-        if let Some(mut path) = ::std::env::home_dir() {
-            path.push("alacritty");
-            path.set_extension("yml");
-            if path.exists() {
-                return Some(path.into());
-            }
+        #[allow(deprecated)]
+        let old = ::std::env::home_dir()
+            .map(|path| path.join("alacritty.yml"));
+        let new = dirs::config_dir()
+            .map(|path| path.join("alacritty\\alacritty.yml"));
+        // TODO: Remove old configuration location warning (Deprecated 03/12/2018)
```

Comment:
I think this comment should be moved from within the method to before the method definition. That should give it a little better visibility so it doesn't get lost.